src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -52,13 +52,35 @@ const ReleaseCommit = React.createClass({
     repository: React.PropTypes.object
   },
 
+  renderMessage(message) {
+    if (!message) {
+      return t('No message provided');
+    }
+
+    let firstLine = message.split(/\n/)[0];
+    if (firstLine.length > 100) {
```

Comment:
First lines in git are expected and anticipated to be 50 characters. Most git UI things use this.